Library/Formula/spaceman-diff.rb

```diff
@@ -0,0 +1,44 @@
+class SpacemanDiff < Formula
+  desc "Diff images from the command line"
+  homepage "https://github.com/holman/spaceman-diff"
+  url "https://github.com/holman/spaceman-diff/archive/v1.0.tar.gz"
+  sha256 "934f38ae863abf15987100062268bcc28d37c39fe8c0a00a0813f07551944632"
+
+  head "https://github.com/holman/spaceman-diff.git"
+
+  depends_on "jp2a"
+  depends_on "imagemagick"
+  depends_on "roundup"
```

Comment:
The upstream README doesn’t list `roundup` as a dependency. Is it really necessary?